### PR TITLE
Update OpenAPI spec and regenerate libs

### PR DESCRIPTION
**Pull request by svix-lucho**

### CODE REVIEW

**Review by svix-james — APPROVED** [review]

File: go/internal/openapi/api_sink.go
Comment (svix-james) [inline go/internal/openapi/api_sink.go:1]: Why is this file deleted?

Comment (svix-lucho) [reply go/internal/openapi/api_sink.go:1]: It's because of the change in tags in the spec, it was basically renamed to `go/internal/openapi/api_stream.go`
